src/components/AppSettingsPage/index.jsx

```diff
@@ -94,44 +116,66 @@ class AppSettingsPage extends React.Component {
               appLogsLink={`/users/${userID}/projects/${projectID}/apps/${appID}/logs/`}
             />
           </div>
-          <div className="MainContentSection">
-            <div className="InformationBarSection">
+          <div className={styles.MainContentSection}>
+            <div className={styles.InformationBarSection}>
               <InformationBar header="Settings" />
             </div>
-            <div className="ContentSection">
-              <div className="DeleteButtonDiv">
+            <div className={styles.ContentSection}>
+              <div className={styles.DeleteButtonDiv}>
                 <PrimaryButton
                   label="Delete App"
-                  className="DeleteBtn"
+                  className={styles.DeleteBtn}
                   onClick={this.showDeleteAlert}
                 />
               </div>
               {openDeleteAlert && (
-                <div className="AppDeleteModel">
+                <div className={styles.AppDeleteModel}>
                   <Modal
                     showModal={openDeleteAlert}
                     onClickAway={this.hideDeleteAlert}
                   >
-                    <div className="DeleteAppModel">
-                      <div className="DeleteModalUpperSection">
-                        <div className="DeleteDescription">
-                          Are you sure you want to delete&nbsp;
-                          <span>{name}</span>
-                          &nbsp; ?
+                    <div className={styles.DeleteAppModel}>
+                      <div className={styles.DeleteModalUpperSection}>
+                        <div className={styles.WarningContainer}>
+                          <div className={styles.DeleteDescription}>
+                            Are you sure you want to delete&nbsp;
+                            <span>{name}</span>
+                            &nbsp;?
+                          </div>
+                          <div className={styles.DeleteSubDescription}>
+                            This will permanently delete the Application.
```

Comment:
@khalifan-kfan I think you can keep this in lower case, what do you think?

> This will permanently delete the application.

Reply:
You mean like.
"this  will permanently delete  the application "
Do you mean that line

Reply:
Alright